### PR TITLE
#867: add a view for comparing schema versions

### LINKED ISSUE #867: Ability to compare topic's schema across versions
### Is your proposal related to a problem?
As a user, I'd like to compare different versions of the specific schema (e.g. the latest with the previous ones)

(Write your answer here.)

### Describe the solution you'd like

(Describe your proposed solution here.)

### Describe alternatives you've considered

(Write your answer here.)

### Additional context

(Write your answer here.)

### PR DESCRIPTION
**Pull request by IndeedSi**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)
I added a view for comparing different schema versions. Fixes #867 

**Is there anything you'd like reviewers to focus on?**
Please check the UI

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [ ] Manually(please, describe, when necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings(e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

**Comment by IndeedSi** [comment]
Probably need to rebase when the redesign is merged :P

**Comment by Haarolean** [comment]
> Probably need to rebase when the redesign is merged :P

Yeah, that's why we were not planning any features for 0.3. 

@GneyHabub this can wait until the redesign is merged in. Up to you.

**Comment by Haarolean** [comment]
@IndeedSi could you rebase please?

**Review by workshur — CHANGES_REQUESTED** [review]
@NelyDavtyan pls rework it using our new styled-components approach

**Review by Motherships — COMMENTED** [review]
Request review when you will be done with changes requested by Oleg, please

File: kafka-ui-react-app/src/components/App.scss
Comment (NelyDavtyan) [reply kafka-ui-react-app/src/components/App.scss:104]: @workshur should I delete this file?

File: kafka-ui-react-app/src/components/common/DiffViewer/__tests__/DiffViewer.spec.tsx
Comment (Motherships) [inline kafka-ui-react-app/src/components/common/DiffViewer/__tests__/DiffViewer.spec.tsx:1]: Do we really need to have `enzyme` in new code when we have issue to get rid of it?

File: kafka-ui-react-app/src/components/Schemas/Diff/Diff.styled.ts
Comment (Motherships) [inline kafka-ui-react-app/src/components/Schemas/Diff/Diff.styled.ts:18]: Direct use of `Colors`

File: kafka-ui-react-app/src/components/Schemas/Diff/Diff.tsx
Comment (Motherships) [inline kafka-ui-react-app/src/components/Schemas/Diff/Diff.tsx:47]: `react-hooks/exhaustive-deps` suggests to add `dispatch` to array of deps

File: kafka-ui-react-app/src/components/Schemas/Details/LatestVersionItem.tsx
Comment (workshur) [inline kafka-ui-react-app/src/components/Schemas/Details/LatestVersionItem.tsx:15]: we already have version without bulma classes.

pls use styled components from this view https://github.com/provectus/kafka-ui/blob/master/kafka-ui-react-app/src/components/Schemas/Details/LatestVersion/LatestVersionItem.tsx

File: kafka-ui-react-app/src/components/common/DiffViewer/__tests__/DiffViewer.spec.tsx
Comment (workshur) [inline kafka-ui-react-app/src/components/common/DiffViewer/__tests__/DiffViewer.spec.tsx:32]: I believe we need to add at least one `expect` to each case

File: kafka-ui-react-app/src/components/Schemas/Details/Details.tsx
Comment (workshur) [inline kafka-ui-react-app/src/components/Schemas/Details/Details.tsx:94]: where are the `leftVersion` and `rightVersion`?

File: kafka-ui-react-app/src/components/Schemas/Diff/Diff.tsx
Comment (Motherships) [inline kafka-ui-react-app/src/components/Schemas/Diff/Diff.tsx:69]: I get why we have `location` in deps

Why we have `history, leftVersion, rightVersion` as deps?

**Review by Motherships — APPROVED** [review]

File: kafka-ui-react-app/src/components/common/DiffViewer/DiffViewer.tsx
Comment (NelyDavtyan) [reply kafka-ui-react-app/src/components/common/DiffViewer/DiffViewer.tsx:1]: will remove it

File: kafka-ui-react-app/src/components/Schemas/Diff/Diff.tsx
Comment (NelyDavtyan) [reply kafka-ui-react-app/src/components/Schemas/Diff/Diff.tsx:69]: you are right, we don't need them, will remove them

**Comment by Haarolean** [comment]
@IndeedSi thank you for your contribution!

**Pull request merged**
